src/PhotosphereLocationSelector.tsx

```diff
@@ -63,6 +72,8 @@ function PhotosphereLocationSelector({
             border: "1px solid black",
             cursor: "crosshair",
             position: "relative",
+            maxWidth: "550px",
+            maxHeight: `${(1 / (navMap.width / 550)) * navMap.height}px`,
```

Comment:
I think it'd be simpler to write `(550 / navMap.width)` instead of `(1 / (navMap.width / 550))`